deploy/base/service.yaml

```diff
@@ -14,11 +14,25 @@ spec:
 apiVersion: v1
 kind: Service
 metadata:
-  name: metrics
+  name: metrics-controller-runtime
   namespace: security-profiles-operator
 spec:
   ports:
-    - port: 80
-      targetPort: 8081
+  - name: https
+    port: 443
+    targetPort: 8442
   selector:
-    app: spod
+    name: spod
```

Comment:
should this point to the manager's label? It would be `name: security-profiles-operator`

Reply:
No, the metrics endpoints are served by the spod only.